albogdan/skule-vote
# Import all Gill Sans weights to be consistent between windows and mac

## Overview

- I'm crying this is the 3rd PR I made on Gill Sans
- #151 and #150 work perfectly for Windows but I just realized it stopped making it work on my mac


## Unit Tests Created

n/a


## Steps to QA

Make sure the fonts look good on your laptop and we gotta push to prod so I can confirm it look's right on my mom's windows :'(
